src/custom/pages/Claim/InvestmentFlow/index.tsx

```diff
@@ -221,6 +223,9 @@ export default function InvestmentFlow({ hasClaims, isAirdropOnly, modalCbs }: I
           ))}
 
           {hasError && <InvestFlowValidation>Fix the errors before continuing</InvestFlowValidation>}
+          {!hasError && someNotTouched && (
+            <InvestFlowValidation>Invest some amount of tokens for each option</InvestFlowValidation>
```

Comment:
I would prefer something more generic than asking people to put money in hehe

```suggestion
            <InvestFlowValidation>Required fields missing</InvestFlowValidation>
```

What do you think @avsavsavs

Reply:
Wdyt of "Investment Amount is required to continue" If its too long, Then I would do "Investment Amount Required"

Reply:
Updated to `Investment Amount is required to continue`